plugins/modules/nd_setup.py

```diff
@@ -248,6 +255,31 @@
                 - The ASN of the BGP peer.
                 type: int
                 required: true
+  deployment_mode:
+    description:
+    - This is used for enabling the available services between
+      Orchestrator, Fabric Controller and Insights during the intial installation.
+    - This option is only applicable for ND version 3.1.1 and later.
+    type: list
+    elements: str
+    choices: [ ndo, ndfc, ndi ]
+  external_services:
+    description:
+    - The persistent Service IPs/Pools to provide
```

Comment:
- The persistent service IPs/Pools to be provided

Reply:
Will fix it.